app/domain/errors.rb

```diff
@@ -365,6 +365,26 @@ module AuthnGcp
       )
     end
 
+    module AuthnJwt
+
+      InvalidIssuerConfiguration = ::Util::TrackableErrorClass.new(
+        msg: "Issuer authenticator configuration is invalid. You should configured as authenticator variables: " \
+              "'{0-resource-name}' or one of the following: '{1-resource-name}','{2-resource-name}'",
+        code: "CONJ00070E"
+      )
+
+      FailedToParseHostnameFromUri = ::Util::TrackableErrorClass.new(
+        msg: "Failed to extract hostname from URI '{0}'",
```

Comment:
should we also log the variable that stores this info?

Reply:
I log it in a previous message:
`
CONJ00054D Issuer value will be taken from 'cucumber:variable:conjur/authn-debug/gitlab8/jwks-uri'
`